src/credentials/activation.ts

```diff
@@ -31,15 +32,21 @@ export async function loginWithMostRecentCredentials(
     toolkitSettings: SettingsConfiguration,
     loginManager: LoginManager
 ): Promise<void> {
+    const profiles = await CredentialsProviderManager.getInstance().getProfiles()
+    const profileNames = Object.keys(profiles)
     const previousCredentialsId = toolkitSettings.readSetting<string>(profileSettingKey, '')
-    if (previousCredentialsId) {
+    if (profiles && profileNames.length === 1) {
+        // Auto-connect if there is exactly one profile, named "default".
+        await loginManager.login(profiles[profileNames[0]])
+        // Toast.
+        vscode.window.showInformationMessage(`Connected to "${profileNames[0]}"`)
```

Comment:
I don't think we need a toast since we already list the connected credentials in the bottom bar. Consider instead having the loading message be "Connecting to ${credentials}..."

Reply:
An explicit notification serves to reduce confusion or expectations around which credentials the toolkit is using.

Reply:
@awschristou What do you think of moving the credential being used to the "Connecting..." message in the explorer tree?

Reply:
I don't believe that separate portions of the toolkit (credentials manager for example) are necessarily initialized and available at that time in order to surface that information.

I think the amount of time that "Connecting..." is visible is small enough that adding the credentials to that label adds much value.

Another benefit of the notification is that the explorer may not be open/visible during toolkit activation.